content/blog/2021-03-10_c2b.Rmd

```diff
@@ -0,0 +1,237 @@
+date: 2021-03-10
+tags:
+  - hospital utilization
+  - actionable intelligence
+  - Excel
+  - Allegheny County
+authors:
+  - ajoshi
+  - roni
+  - cscott
+heroImage: /blog/images/blog-lg-c2b.jpg # size: 1120x440 jpg and webp format
+heroImageThumb: /blog/images/blog-thumb-c2b-1.jpg # size: 300x200 jpg and webp format
+# related:
+summary: | 
+  In our partnership with the Allegheny County Health Department, 
+  we explored avariety of solutions to different pressing challenges 
+  during the pandemic. Cases2Beds was one such tool that came as a 
+  result of rising COVID cases in Allegheny County. 
+  
+  The goal of Cases2Beds is to project likely hospital bed 
+  utilization from any given set of lab-confirmed cases. The intuition of 
+  Cases2Beds is that some fraction ((hospitalization rate) of individuals who 
+  test positive today will be hospitalized after a time delay 
+  (from test date to admit date) and for a varying duration 
+  (from admit date to discharge date). These values and distributions vary by 
+  individual traits (primarily age group, and then race or gender). 
+  Using the current relevant hospitalization rates, delay distributions,
+  and duration distributions, we show what bed usage might be for the inputted 
+  positive cases. We are not aware of other generative approaches for 
+  hospital utilization.
+  
+acknowledgements: |
+ Thank you to the Allegheny County Health Department (especially Antony Gnalian,
+ Dr. LuAnn Brink, and Dr. Debra Bogen) for their invaluable feedback, efforts, and
+ shared interest in actionable intelligence. 
+ 
+ Many members of the Delphi Group, including 
+ Sumit Agrawal, Katie Mazaitis, 
+ and Phil McGuinness met regularly with the Allegheny County 
+ Health Department and investigated various solutions other than Cases2Beds. 
+ Andrew Chin and Benjamin Smith investigated Cases2Beds alternatives.
+ 
+ Zachary Lipton, Helen Zhou, and Cheng Cheng 
+ provided historical data from Florida and valuable support. 
+ 
+ Spencer Whitman, Phil McGuinness, and Sam Gratzl helped edit and format 
+ this blog post. 
+output:
+  blogdown::html_page:
+    toc: true
+---
+
+One of the Delphi Group's goals is to create informative tools for healthcare organizations. Tools are only useful, however, if the insights they provide can inform concrete actions.  That is to say, these tools must provide actionable intelligence.  To address local challenges during the pandemic, we partnered with the Allegheny County Health Department (ACHD). One of our shared worries in early November 2020, as the case rates in the region continued to increase, was if hospitals located in the county would run out of hospital beds for COVID patients (Fig 1). 
+
+```{r WPRDC, message=FALSE, warning=FALSE, fig.width=9, out.extra = 'class="wide-figure"'}
+library(ggplot2)
+library(scales)
+library(dplyr)
+library("ggpubr")
+library(scales)
+
+full_df <- tibble(read.csv(file="../../public/blog/2021-03-10_c2b/test_and_hosp.csv"))
```

Comment:
I thought I moved the files and updated the links in my update PR...

Reply:
Hm I wonder what happened here, I'll undo my last commit

Reply:
Merged. Please take a look.